js/kucoinfutures.js

```diff
@@ -1504,8 +1504,10 @@ module.exports = class kucoinfutures extends kucoin {
         //    }
         //
         const data = this.safeValue (response, 'data');
-        const fundingTimestamp = this.safeNumber (data, 'timePoint');
-        // the website displayes the previous funding rate as "funding rate"
+        const previousFundingTimestamp = this.safeInteger (data, 'timePoint');
+        const granularity = this.safeInteger (data, 'granularity');
+        const fundingTimestamp = this.sum (previousFundingTimestamp, granularity);
+        // the website displays the previous funding rate as "funding rate"
```

Comment:
could you move this comment up to under `const previousFundingTimestamp = this.safeInteger (data, 'timePoint');`